packages/playground/src/dashboard/components/node_details.vue

```diff
@@ -15,11 +15,11 @@
     </div>
   </td>
 
-  <td :colspan="columnsLen" v-else>
+  <td :style="{ backgroundColor: 'rgb(var(--v-theme-background))'}" :colspan="columnsLen" v-else>
     <v-row class="ma-2">
       <v-col :cols="getColSize">
         <div class="mt-3">
-          <card-details
+          <card-details :style="{ backgroundColor: 'rgb(var(--v-theme-background))'}"
```

Comment:
For card-details style Can we add a scoped style to all of them in this component? instead of repeating the style line